### ISSUE TITLE
  it's a bit troublesome to install on apple systems. error: externally-managed-environment......

### ISSUE THREAD

**Issue by 69567272** [apache/incubator-fury#1638]
According to the documentation, I ran the following two commands, but the first one reported an error, how to solve it?
I'm new here. I'm a little confused.
1、pip install pyarrow==14.0.0
Error: externally-managed-environment
× This environment is externally managed
╰─> To install Python packages system-wide, try brew install
    xyz, where xyz is the package you are trying to
    install.
    
    If you wish to install a Python library that isn't in Homebrew,
    use a virtual environment:
    
    python3 -m venv path/to/venv
    source path/to/venv/bin/activate
    python3 -m pip install xyz
    
    If you wish to install a Python application that isn't in Homebrew,
    it may be easiest to use 'pipx install xyz', which will manage a
    virtual environment for you. You can install pipx with
    
    brew install pipx
    
    You may restore the old behavior of pip by passing
    the '--break-system-packages' flag to pip, or by adding
    'break-system-packages = true' to your pip.conf file. The latter
    will permanently disable this error.
    
    If you disable this error, we STRONGLY recommend that you additionally
    pass the '--user' flag to pip, or set 'user = true' in your pip.conf
    file. Failure to do this can result in a broken Homebrew installation.
    
    Read more about this behavior here: <https://peps.python.org/pep-0668/>

note: If you believe this is a mistake, please contact your Python installation or OS distribution provider. You can override this, at the risk of breaking your Python installation or OS, by passing --break-system-packages.
hint: See PEP 668 for the detailed specification.

2、bazel build //cpp/fury/row:fury_row_format

Why not brew install pyarrow?

**Comment by chaokunyang** [comment]
Yes, you can install pyarrow by brew. It's a comon practice to install python packages using something like conda/pyenv to resolve package conflicts

**Comment by 69567272** [comment]
brew search arrow
==> Formulae
apache-arrow 
  Is this it?

**Comment by esafak** [comment]
Is this solved? I use macports: `sudo port install py313-pyarrow` or `sudo port install apache-arrow`

**Comment by 69567272** [comment]
> Is this solved? I use macports: `sudo port install py313-pyarrow` or `sudo port install apache-arrow`解决了吗？我使用macports：`sudo port install py313-pyarrow`或`sudo port install apache-arrow`

Hi, There's no solution. We've got another solution.

**Comment by esafak** [comment]
So we can close the ticket? I don't understand exactly.
